docs/css/animation-book.css

```diff
@@ -0,0 +1,99 @@
+/* 表紙 */
+.book-simple {
+    position: relative;
+    box-sizing: border-box;
+    width: 90%;
+    height: 90%;
+    margin-right: auto;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    margin-left: auto;
+    padding-bottom: 10px;
+    -webkit-transition: box-shadow 2s ease;
+            transition: box-shadow 2s ease;
+    background-color: #124f4b;
+    box-shadow: -2px 0 0 0 #124f4b,-1px 4px 1px 1px rgba(0,0,0,.2),20px 0 20px 0 rgba(0,0,0,0);
+  
+    -webkit-perspective: 1000px;
+            perspective: 1000px;
+    -webkit-perspective-origin: 0 150%;
+            perspective-origin: 0 150%;
+  }
+  .book-simple:hover, #book-simple-open:checked ~ #book-simple-contents {
+    box-shadow: -2px 0 0 0 #124f4b,0 4px 1px 1px rgba(0,0,0,.2),-195px 5px 1px -5px rgba(0,0,0,.2);
+  }
+  .book-simple::before {
+    position: absolute;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    display: block;
+    width: 100%;
+    height: calc( 100% - 10px);
+    content: '';
+    -webkit-transition-timing-function: ease;
+            transition-timing-function: ease;
+    -webkit-transition-duration: 2.5s;
+            transition-duration: 2.5s;
+    -webkit-transition-property: transform;
+            transition-property: transform;
+    -webkit-transform-origin: 0 0;
+            transform-origin: 0 0;
+    background-color: #249690;
+    box-shadow: 0 2px 0 0 #124f4b;
+  }
+  .book-simple:hover::before, #book-simple-open:checked ~ #book-simple-contents::before {
+    -webkit-transform: rotateY(-185deg);
+            transform: rotateY(-185deg);
+  }
+  
+  /* 中身 */
+  .book-simple .book-inbox {
+    position: relative;
+    box-sizing: border-box;
+    height: 100%;
+    padding: 10px;
+    transition: box-shadow 1.8s ease;
+    transition-delay: .3s;
+    background-color: #e2e1dc;
+    box-shadow: 200px 0 1px 0 rgba(0,0,0,.4) inset,-2px 0 0 0 #249690;
+  }
+  .book-simple .book-inbox:after {
+    position: absolute;
+    bottom: -8px;
+    left: 0;
+    display: block;
+    box-sizing: border-box;
+    width: 100%;
+    height: 8px;
+    content: '';
+    background-color: #afada3;
+  }
+  .book-simple:hover .book-inbox, #book-simple-open:checked ~ #book-simple-contents .book-inbox {
+    box-shadow: -0 0 8px 0 rgba(0,0,0,.0) inset,-2px 0 0 0 #249690;
+  }
+  
+  /* ボタン */
+  .book-simple-switch {
+    display: block;
+    width: 6em;
```

Comment:
ボタンの文字が改行されているのが気になるので、これぐらいあるといいかも
単位がここだけ `em` なのが気になるので、 130px とかでもいいかも

```suggestion
    width: 8em;
```

![スクリーンショット 2021-04-28 20 33 49](https://user-images.githubusercontent.com/35416089/116397130-29b5b380-a861-11eb-9592-5973ed444a50.png)

Reply:
（いいかもしか書いてない…）

Reply:
そうやんねー。１３０センチにするわー